app/topics/topic-management/delete-topic/delete-topic.component.html

```diff
@@ -0,0 +1,20 @@
+<md-card class="default-card">
+  <md-card-title>{{ 'delete topic' | translate }}</md-card-title>
+  <md-card-content>
+    <div *ngIf="!isDeleted">
+      <div>{{ 'confirm delete' | translate }}</div>
+      <div class="topic-title">{{ topic.title }}</div>
+      <div class="actions">
+        <button md-raised-button color="primary" [routerLink]="['/topics', topic.id]">{{ 'cancel' | translate }}</button>
+        <button md-raised-button color="warn" [disabled]="!isReady" (click)="deleteTopic()">{{ 'delete topic' | translate }}</button>
+      </div>
+    </div>
+    
+    <div *ngIf="isDeleted">
+      <div><span class="topic-title">{{ topic.title }}</span> {{ 'deleted' | translate }}</div>
+      <div class="actions">
+        <button md-raised-button color="primary" [routerLink]="['/my-topics']">{{ 'back to my topics' | translate }}</button>
```

Comment:
I don't think we need that extra page / div. When the topic is deleted successfully an automatic redirect to My Topics should be enough. The message that it is deleted successfully is shown by the toaster, so the user already got feedback and this page is just an (annoying) extra click for the user.